packages/core/content-manager/admin/src/pages/EditView/components/Header.tsx

```diff
@@ -390,7 +390,7 @@ ConfigureTheViewAction.type = 'configure-the-view';
  * Because the icon system is completely broken, we have to do
  * this to remove the fill from the cog.
  */
-const StyledCog = styled(Cog)`
+const StyledListPlus = styled(ListPlus)`
```

Comment:
this can probably be removed and you directly call `ListPlus` because the svg default colors are `currentColor`